client/ayon_flame/plugins/publish/collect_shots.py

```diff
@@ -0,0 +1,342 @@
+
+        # try to find attributes
+        attributes = {
+            "xml_overrides": {
+                "pixelRatio": 1.00}
+        }
+        # search for `:`
+        for split in self._split_comments(comment):
+            # make sure we ignore if not `:` in key
+            if ":" not in split:
+                continue
+
+            self._get_xml_preset_attrs(
+                attributes, split)
+
+        # add xml overrides resolution to instance data
+        xml_overrides = attributes["xml_overrides"]
+        if xml_overrides.get("width"):
+            attributes.update({
+                "resolutionWidth": xml_overrides["width"],
+                "resolutionHeight": xml_overrides["height"],
+                "pixelAspect": xml_overrides["pixelRatio"]
+            })
+
+        return attributes
+
+    def _get_xml_preset_attrs(self, attributes, split):
+
+        # split to key and value
+        key, value = split.split(":")
+
+        for attr_data in self.xml_preset_attrs_from_comments:
+            a_name = attr_data["name"]
+            a_type = attr_data["type"]
+
+            # exclude all not related attributes
+            if a_name.lower() not in key.lower():
+                continue
+
+            # get pattern defined by type
+            pattern = TXT_PATTERN
+            if a_type in ("number", "float"):
+                pattern = NUM_PATTERN
+
+            res_goup = pattern.findall(value)
+
+            # raise if nothing is found as it is not correctly defined
+            if not res_goup:
+                raise ValueError((
+                    "Value for `{}` attribute is not "
+                    "set correctly: `{}`").format(a_name, split))
+
+            if "string" in a_type:
+                _value = res_goup[0]
+            if "float" in a_type:
+                _value = float(res_goup[0])
+            if "number" in a_type:
+                _value = int(res_goup[0])
+
+            attributes["xml_overrides"][a_name] = _value
+
+        # condition for resolution in key
+        if "resolution" in key.lower():
+            res_goup = NUM_PATTERN.findall(value)
+            # check if axpect was also defined
+            # 1920x1080x1.5
+            aspect = res_goup[2] if len(res_goup) > 2 else 1
+
+            width = int(res_goup[0])
+            height = int(res_goup[1])
+            pixel_ratio = float(aspect)
+            attributes["xml_overrides"].update({
+                "width": width,
+                "height": height,
+                "pixelRatio": pixel_ratio
+            })
+
+    def _split_comments(self, comment_string):
+        # first split comment by comma
+        split_comments = []
+        if "," in comment_string:
+            split_comments.extend(comment_string.split(","))
+        elif ";" in comment_string:
+            split_comments.extend(comment_string.split(";"))
+        else:
+            split_comments.append(comment_string)
+
+        return split_comments
```

Comment:
Requires `import re`.
```suggestion
        return re.split(r",|;", comment_string)
```